src/oc_erchef/priv/reindex-opc-piecewise

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env escript
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92-*-
+%% ex: ts=4 sw=4 et
+
+%% Copyright (c) 2020 Chef Software, Inc.
+%% All Rights Reserved
+
+%% TODO: The cookie used by erchef should be part of config
```

Comment:
Any plans to pick these up in another PR?

Reply:
Perhaps not immediately. I think these come from when the escripts were originally created.